src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: 'Harry Potter',
+      author: 'J.K Rolling',
+      category: 'fictional',
+    },
+  ],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
```

Comment:
- [ ] Great job with your project, just a small change, please following the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md), you should have an empty array as initial state in your book and categories slices.
   ![image](https://github.com/anshuman7negi/book-store/assets/107952724/7fca8e42-e545-4dd6-ad10-ebbe31d78447)